docs/copied-from-beats/shared-logstash-config.asciidoc

```diff
@@ -14,7 +14,7 @@
 To send events to {ls}, you also need to create a {ls} configuration pipeline
 that listens for incoming Beats connections and indexes the received events into
 {es}. For more information, see the section about
-{stack-ov}/get-started-elastic-stack.html#logstash-setup[configuring {ls}] in
+{stack-gs}/get-started-elastic-stack.html#logstash-setup[configuring {ls}] in
```

Comment:
This is not defined anywhere.

Reply:
That https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L27 is the reason this is working - so all good.